.travis.yml

```diff
@@ -31,6 +31,9 @@ before_install:
 
 install:
   - pip install $DJANGO
+  # Temporary: remove when openwisp-utils#71 is merged
+  - pip install "https://github.com/waleko/openwisp-utils/tarball/issues/40" --upgrade
```

Comment:
I released openwisp-utils 0.4.1 which contains your patch https://github.com/openwisp/openwisp-utils/pull/71.
You can now remove this temporary addition.